plugins/00_dokku-standard/commands

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
-source "$PLUGIN_PATH/common/functions"
-source "$PLUGIN_PATH/config/functions"
+source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+source "$PLUGIN_CORE_AVAILABLE_PATH/config/functions"
```

Comment:
If you want to guarantee you're getting a core plugin regardless of it being enabled or not, yes. (side note: while dokku won't let you disable a core plugin, plugn is like a honeybadger, it don't care)